src/main/java/dev/meyi/bn/BazaarNotifier.java

```diff
@@ -47,10 +49,16 @@
   public static JSONObject bazaarCache = new JSONObject();
   public static JSONArray bazaarDataFormatted = new JSONArray();
 
+
   public static JSONObject bazaarConversions = new JSONObject(
-      new JSONTokener(BazaarNotifier.class.getResourceAsStream("/bazaarConversions.json")));
+      new JSONTokener(Objects.requireNonNull(BazaarNotifier.class.getResourceAsStream("/bazaarConversions.json"))));
   public static JSONObject bazaarConversionsReversed = new JSONObject(
-      new JSONTokener(BazaarNotifier.class.getResourceAsStream("/bazaarConversionsReversed.json")));
+      new JSONTokener(Objects.requireNonNull(BazaarNotifier.class.getResourceAsStream("/bazaarConversionsReversed.json"))));
+  public static JSONObject config = new JSONObject(
+          new JSONTokener(Objects.requireNonNull(BazaarNotifier.class.getResourceAsStream("/config.json"))));
```

Comment:
In the future, we can use this file to hold default values instead of in that other class. The user's config is already being saved, but it isn't really easy to access in its current form.